libraries/Microsoft.Bot.Builder.AI.QnA/Dialogs/QnAMakerDialog.cs

```diff
@@ -290,6 +290,24 @@ public QnAMakerDialog([CallerFilePath] string sourceFilePath = "", [CallerLineNu
         [JsonProperty("rankerType")]
         public StringExpression RankerType { get; set; } = new StringExpression(RankerTypes.DefaultRankerType);
 
+        /// <summary>
+        /// Gets or sets a value indicating whether gets or sets enablePreciseAnswer. 
+        /// </summary>
+        /// <value>
+        /// A value indicating whether to enable Precise Answer or not. 
+        /// </value>
+        [JsonProperty("enablePreciseAnswer")]
+        public bool EnablePreciseAnswer { get; set; }
+
+        /// <summary>
+        /// Gets or sets a value indicating whether gets or sets displayPreciseAnswerOnly. 
```

Comment:
Grammar is incorrect. Please copyedit.

Reply:
Content is not helpful. Please significantly improve

Reply:
Fixed.

---
In reply to: [430548583](https://github.com/microsoft/botbuilder-dotnet/pull/3935#discussion_r430548583) [](ancestors = 430548583)

Reply:
Modified content as suggested.

---
In reply to: [430548743](https://github.com/microsoft/botbuilder-dotnet/pull/3935#discussion_r430548743) [](ancestors = 430548743)